data/hai/hai missions.txt

```diff
@@ -1808,3 +1846,442 @@ mission "Nanachi Meeting"
+			branch signlanguage
+				has "Remnant: Learn Sign Follow Up: offered"
+			`	"Greetings, Captain <first>." The stranger chants. "My name is Rayn, and my colleague said you had a request from another faction. How can I help?"`
+				goto stridercontinue
+			label signlanguage
+			`	"Greetings, Captain <first>." The stranger signs. "My name is Rayn, and my colleague said you had a request from another faction. How can I help?"`
+			label stridercontinue
+			choice
+				`	"The Hai are hoping to learn from someone."`
+				`	"The Hai need help defending their space."`
+			`	"So the Hai want to improve their military technology? I presume this is in response to their long-standing war against the Unfettered?"`
+			choice
+				`	"Yes, although they don't think of it as such."`
+				`	"It isn't a war; it's a defensive action."`
+			`	Rayn looks amused. "It does not matter what they call it. A war by any name is just as bloody. But that aside, how do they know about us to make such a request?"`
+			choice
+				`	"They asked if I knew anyone who had in-flight repair technology, and I said I knew a group of human researchers."`
+				`	"They were looking for someone who had repair technology, and I said I could find someone."`
+			`	"So they do not actually know about us. That is good." Rayn looks thoughtful and closes his eyes for a moment, then continues. "Pick up the Hai diplomats, but do not tell them where you are going, and take them to these co-ordinates on Deadman's Cove in the Almach system. It is a location in an uninhabited area where we will be able to talk. We will be waiting with something that might be able to help them - if they can meet our price."`
+	on complete
+		conversation
+			`Shortly after landing on <planet>, you spot Ambassador Yashili's aide rushing across the tarmac to meet you. "Please tell me they agreed. They did, right?" You barely have the chance to nod before the aide continues on. "The Ambassador will be ready in just a moment. She will meet you in the starport shortly. Thank you!" And with that, the officious Hai scurries off back the way it came.`
+
+
+
+mission "Strider: Remnant 2"
+	name "Contact the Remnant"
+	description "Transport Ambassador Yashili and her delegation to <destination> to attempt to obtain hull repair technology from the Remnant."
+	source "Hai-home"
+	destination "Deadman's Cove"
+	infiltrating
+	passengers 3
+	blocked "The Hai are sending a delegation to <destination> and will require 3 bunks for the trip."
+	to offer
+		has "Strider: Remnant 1: done"
+	on offer
+		conversation
+			`You enter the spaceport and find the Hai delegation meant to travel to the Remnant to negotiate for the hull repair technology. The leader of the delegation introduces herself as Ambassador Yashili. "I heard that we will be travelling a great distance across the galaxy. May we have a safe journey."`
```

Comment:
We already had the introduction to Yashili in `Strider: Remnant 1` so this is redundant.